neasqc_wp61/models/quantum/embedder.py

```diff
@@ -126,7 +135,7 @@ def _add_embeddings_to_dataset(
                 "compute_embeddings must be called before add_embeddings_to_dataset"
             )
         if not isinstance(embeddings, list):
```

Comment:
You can't have code + a raiseNotImplemented error.
Either you DO want your code to be run, in which case you need to call super, and you don't want raisNotImplemented to bet there! 
Or you you DON'T want your code to be run, and you raise a NIE.

Here in your code what's supposed to happen to if not isinstance???